Add description of File Annotation addition (#8)
ome/omero-guide-introduction: docs/annotate.rst

Annotate Data and Filter using Annotations
==========================================

> There are several ways of how to add annotations to objects in OMERO. Here, addition and filtering of annotations using OMERO.web is described. You can add annotations using the OMERO.web interface to any object (or more of them) which you can select in the left-hand side tree or central pane, this means ``Project``, ``Dataset``, ``Image``, ``Screen``, ``Plate`` and ``Well``.

Reviewer: ```suggestion
There are several ways to add annotations to objects in OMERO. Here, addition and filtering of annotations using OMERO.web is described. You can add annotations using the OMERO.web interface to any object(s) that you can select in the left-hand-side tree or central pane, this means ``Project``, ``Dataset``, ``Image``, ``Screen``, ``Plate`` and ``Well``.
```

#. Connect using the provided credentials

#. Let us add annotations to Images.

#. Note that you can add annotations using this interface to any object (or more of them) which you can select in the left-hand side tree or central pane, this means ``Project``, ``Dataset``, ``Image``, ``Screen``, ``Plate`` and ``Well``.

> #. First, we will add Tags to indicate Metaphase stages of these cells.

Reviewer 2: For " Step-by-step" instructions each step should involve something the user should do. Steps 3 and 4 don't, so maybe combine them with step 5? Or move some of the information to the introduction?

Author: fixed in https://github.com/ome/omero-guide-introduction/pull/8/commits/2255c4e0962df4ace84b96ce83fb35a38713876d

> 

Reviewer 2: > Choose the Tag option in the right-hand ``General`` tab and click ``[ + ]`` to launch the Tag dialog

- `Choose the Tag option` should be `Choose the Tags option`
- `launch the Tag dialog` should perhaps be `launch the Tags Selection dialog` if you want to match the dialog title?

> Choose the existing Metaphase

`Choose the existing ``Metaphase`` tag`?

Author: fixed in https://github.com/ome/omero-guide-introduction/pull/8/commits/3bf56ba5d06518443193797d8206c542bbc96a39

>    - Click ``>`` to move it to the right column, then click ``OK``.

Reviewer 2: > click ``OK``

Do you mean `Save`?

![image](https://user-images.githubusercontent.com/1644105/88656192-d511bc00-d0c7-11ea-9fe8-a249246df0a6.png)

Author: fixed in https://github.com/ome/omero-guide-introduction/pull/8/commits/ab265a5581b93048dca8fb7592ee2254041d50ac

#. Let us now add Key-Value Pairs

>    - Select an Image from the Dataset and in the right-hand pane in ``General`` tab, click the harmonica Key-Value Pairs.

Reviewer 2: Earlier in this guide it says "Choose the Tag option in the right-hand General tab", here it calls it "the harmonica", and also below. I think the terminology should be consistent.

Author: fixed in https://github.com/ome/omero-guide-introduction/pull/8/commits/b8b2e91d0977eb0b942dd7bd7fc37e5f39bcadd1

   - The Key-Value Pairs allow you to add lab-book-like additional metadata for the Image. These Key-Value Pairs are also specifically searchable. See :doc:`search-omero`.

#. For adding of File Attachments:

   - Select one or more objects in the left-hand side tree, such as ``Dataset`` or ``Image``.
   - Expand the ``Attachments`` harmonica in the right-hand pane. 
   - Click the plus button.

      |image1b|

   - You can attach any type of file using this function. If you select a file from your local filesystem using the ``Browse`` button, the feature will upload that file to the OMERO.server and save it there. The content of ``.pdf``, ``CSV`` and plain text files is also searchable in OMERO.

#. Remove a File Attachment.

   - Find the File Attachment you have just added.

   - Click on the ``minus`` sign to the right of it.

      |image1c|

>    - The removal action just unlinks the File Attachment from the particual selected object(s). The File Attachment is not deleted from the server. If deletion is needed, click in the workflow above on the ``cross`` icon instead of the ``minus`` icon.

Reviewer: ```suggestion
   - The removal action just unlinks the File Attachment from the selected object(s). The File Attachment is not deleted from the server. If deletion is needed, click in the workflow above on the ``cross`` icon instead of the ``minus`` icon.
```

   - The Key-Value Pairs allow you to add lab-book-like additional metadata for the Image. These Key-Value Pairs are also specifically searchable. See :doc:`search-omero`.

#. For adding of File Attachments:

   - Select one or more objects in the left-hand side tree, such as ``Dataset`` or ``Image``.
   - Expand the ``Attachments`` harmonica in the right-hand pane. 
   - Click the plus button.

      |image1b|

>    - You can attach any type of file using this function. If you select a file from your local hierarchy, the feature will upload that file to the OMERO.server and save it there. The content of ``.pdf``, ``CSV`` and plain text files is also searchable in OMERO.

Reviewer 2: > from your local hierarchy

Do you mean from the local filesystem using the `Browse...` button?

Author: fixed in https://github.com/ome/omero-guide-introduction/pull/8/commits/fa082fdac0f6da738107403d5e2b9037d462a865
